packages/components/src/Menu/__tests__/Menu.test.tsx

```diff
@@ -308,7 +308,7 @@ describe("Menu", () => {
   });
 });
 
-it("should focus first action item from the menu when activated", async () => {
+it("should focus first action item from the menu when activated by keyboard", async () => {
```

Comment:
with focus trap gone, we no longer focus the first item when opening with clicks, only keyboard interactions cause that